### ISSUE TITLE
Could you provide pretrained models?

### ISSUE THREAD

**Issue by Fivethousand5k** [batmanlab/vessel-flow-consistency-ssl#2]
Hello! Thanks for your awesome and inspiring work. Could you provide pretrained models for evaluation?

**Comment by rohitrango** [comment]
Hi, thank you for your interest in our work! 

Unfortunately, I looked around and it seems like I've misplaced the pretrained models, and cannot find them. However, the config files I mentioned in the README are the ones used to generate the results mentioned in the paper. They shouldn't take too long to run either. 

If you need help with anything else, please let me know. Thanks.

**Comment by Fivethousand5k** [comment]
Hello, I tried to follow yours scripts to train the models on Drive and TubeTK datasets. However, the training duration seems to be too long, especially for the TubeTK(3D dataset).  It almost took me a dozen of hours to finish **an** epoch with 2 Titan RTX 24G.  I don't believe this cost acceptable.  Am I wrong somewhere？How is about your training？ BTW，from my observation, I believe it is highly related to the dense loss computation in the training process, and find that maybe the loss computation is limited within a single GPU (High memory occupation merely in the GPU 0).
